src/k8s-engine/images/imageProcessor.ts

```diff
@@ -379,6 +379,8 @@ export abstract class ImageProcessor extends EventEmitter {
 
   abstract deleteImage(imageID: string): Promise<childResultType>;
 
+  abstract deleteImages(images: string[]): Promise<childResultType>;
```

Comment:
These look more like `imagesIDs` to me than `images` (which should usually refer to the image object, not a string).